knot-thing/src/main/java/com/cesar/knot_sdk/knot_messages/KNoTThingRegister.kt

```diff
@@ -0,0 +1,15 @@
+package com.cesar.knot_sdk.knot_messages
+
+/**
+ * This class is an abstraction for the registration message. Its property names
+ * are already mapped to the expected values in the message json, which means
+ * it's really easy to convert these values to the expected json message.
+ * @property id a unique identifier for the Thing, be careful to not generate a
+ * duplicate name, as the KNoT-Protocol does not define what happens when
+ * receiving a duplicate id.
+ * @property name a string that represents this device in some manner.
+ */
+data class KNoTThingRegister(
```

Comment:
Is not clear to me that this class is an abstraction for the registration message by looking only to the name.

Reply:
Done.